assets/build/api/openapi-config.ts

```diff
@@ -49,7 +49,7 @@ const config: ConfigFile = {
       filterEndpoints: pathMatcher(/schedule/i)
     },
     '../../js/src/core/modules/element/element-api-slice.gen.ts': {
-      filterEndpoints: pathMatcher(/path/i)
+      filterEndpoints: pathMatcher(/(path|delete)/i)
```

Comment:
Isn't `delete` a bit dangerous as filter here? Should we be more specific so that we do not accendentially add wrong endpoints?

Reply:
fixed with: https://github.com/pimcore/studio-ui-bundle/pull/564/commits/9b63212fbb3cb21983254d523e41ed50bf9c8033